src/guc.c

```diff
@@ -437,6 +438,26 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	/*
+	 * Define the limit on number of invalidation-based refreshes we allow per
+	 * refresh call. If this limit is exceeded, fall back to a single refresh that
+	 * covers the range decided by the min and max invalidated time.
+	 */
+	DefineCustomIntVariable(MAKE_EXTOPTION("materializations_per_refresh_window"),
```

Comment:
Why not use the same global variable name `cagg_max_individual_materializations` ??? Also wondering if it should be a cagg option instead of a session guc because doing that we can have different values for different caggs and policies will use it.

Reply:
I used the name of the existing setting. Not sure if customers are actively using it, but I wanted to be on the safe side and compatible. 

If we decide that this is a useful setting for users to tune, we should convert it into a CAgg setting. However, this would be a breaking change and I would do this after we have confirmation that it makes sense to actually change it (maybe we decide to re-work this algorithm entirely and we would need other settings).